metadata/UiPathDocumentOCR_CPU__1__metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+  "changeLog": "Model: 22.2.4",
+  "cpu": 0,
+  "description": "Machine Learning model(available in Preview) for extracting text and checkboxes/radio buttons from Documents. Please see more details including supported languages and link to Activities guide in the Document Understanding Guide here: https://docs.uipath.com/document-understanding/v2021.4/docs/ml-packages#uipath-document-ocr. For alternative deployment methods in high-volume scenarios see also this page: https://docs.uipath.com/document-understanding/v2021.4/docs/ocr-services",
+  "displayName": "UiPathDocumentOCR_CPU",
+  "gpu": 0,
+  "inputDescription": "ML Skills deployed using this ML Package can be integrated into RPA workflows using the UiPath Document OCR activity from OCR.Activities pack in the Official feed. File formats accepted include pdf, tiff, jpg or png files. In non-airgapped deployments, the activity requires the Document Understanding API Key input which you need to obtain from your UiPath Automation Cloud account, in the Licenses -> Other Services view.",
+  "inputType": "JSON",
+  "memory": 0,
+  "mlPackageLanguage": "PYTHON37_DU",
+  "name": "UiPathDocumentOCR_CPU",
+  "outputDescription": "Please refer to the documentation of the Activity used to query the ML Skill.",
+  "processorType": "CPU",
+  "projectId": "[project-id]",
+  "retrainable": false,
+  "stagingUri": "[staging-uri]",
+  "projectName": "UiPath Document Understanding",
+  "projectDescription": "UiPath models to classify and extract information from images and pdfs.",
+  "tenantName": "UiPath",
+  "minAIFabricVersion": "v21.10",
+  "languageVersion": 3,
```

Comment:
@irinel-birsan-uipath  i thought we were introducing version 4 for new versions of models. Is that postponed ?

Reply:
@mohitseth-nitrr This is an off-band release for existing SF